build.gradle

```diff
@@ -73,14 +73,14 @@ dependencies {
     google()
   }
 
-  api 'com.segment.analytics.android:analytics:4.3.1'
-  api 'com.comscore:android-analytics:5.7.2'
+  api 'com.segment.analytics.android:analytics:4.5.0-beta.0'
```

Comment:
@breel93 - one last item: we should leave the Analytics dep at the latest stable dependency, which is still 4.3.1. If you bump the Segment library dep to `4.5.0-beta.0` in the Comscore dep, then only users with that version or higher of Segment's lib will be able to use Comscore.